workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
 ---
-Overview
---------
-Container Monitoring provides real-time visibility into health and performance of containerized environments. 
 
-Container List
---------------
-The container list enables visibility into all containers across your environment excluding Kubernetes. It provides a detailed overview of your containers' health, resource consumption, and deployment. The collection of containers is container id, container name, host, status, cpu usage, ram usage, etc.
+This guide provides an overview of Middleware’s Infrastructure Monitoring page. Effortlessly collect, configure, and visualize complex telemetry data with our autodetection capabilities across your entire Infrastructure stack.
 
-![](/images/infra-monitoring/c1.png)
+Quickly isolate and drill into blockers within your teams Infrastructure stack, monitor your cluster, node, and pod data in one place, and effortlessly manage and review your overall Infrastructure utilization cluster by cluster.
 
-### Container ID
-This shows the id of the container and IDs are randomly generated and are typically 12 characters long.
+# Key Concepts
 
-### Container Name
-This shows the name of the container and the name is a user-friendly identifier that is assigned to a container.  
+### Host List
 
-### Host
-The host can be a physical or virtual machine running any operating system, such as Linux, Windows, or macOS.  
+Host List provides insight into your Kubernetes orchestration data, database usage, system-wide infrastructure utilization and more. It is the highest level compoenent overview, providing a top-down approach for diagnosing, monitoring, and maintaining your stack.
 
-### Status
-The container status refers to the current state of a container. The container status states include:
+### Containers
 
--   **Running**: The container is currently running and its main process is active.
--   **Restarting**: The container is in the process of restarting after being stopped or restarted.
--   **Exited**: The container has stopped running and its main process has exited.
--   **Dead**: The container has stopped running due to an error or crash, and its main process is no longer active.
+Containers provide visibility into the health and performance of your containerized environments. This section does not include containers that are part of an orchestration tool (e.g. Kubernetes). 
 
-### CPU Usage
-This shows the percentage of CPU used. The CPU usage of a container refers to the percentage of the host system's CPU resources that are currently being used by the container's processes.
+### Processes
 
-### RAM Usage
-This shows the percentage of RAM used. The container RAM usage in percentage refers to the proportion of allocated memory resources that a container is currently using on the host system.
+Processes are managed within your containers to ensure your Infrastructure is effectively monitored, maintained, and optimized for performance, reliability, and operability.
```

Comment:
Making a point here that helps with the other parts of this section as well. You're accidentally blending what a thing is (in this case "Processes") with what a user can do with the data. A good way to make sure that doesn't happen is to structure the sentence as:

What the thing is. How we get the data. What the section is for.

For example,

"Processes are system level methods being run or managed inside of your hosts. This data is pulled directly from the kernel of each host and imported to Middleware through the Host Agent. You may use this data to analyze process health, overall process impact on your infrastructure, and drill into each process to uncover further data points"

It's worth checking with eng exactly _how_ this data comes to us as I mostly fabricated that portion above just for the example.

Reply:
also checking on this with Tejas

Reply:
"....system level programs being executed inside of your hosts or containers."

Skip the word kernel:  "...pulled directly from the operating system and imported to MW through..."